### PR TITLE
portfolio includes black thursday and enigma re-evaluation

### PR DESCRIPTION
**Pull request by chadellison**

### CODE REVIEW

**Comment by worace** [comment]
@chadellison you need to update this PR by pulling from the upstream master

File: students/1511/chads_portfolio.md [hunk 1]
```diff
@@ -175,10 +224,10 @@ reflecting on what is missing here and hopefully be able to tweak your process i
 * would like to see you getting involved with extra-curric student groups and activities -- think your personality
 could help give momentum to these kind of things and get them off the ground
 
-### Outcome - PROVISIONAL PASS
+### Outcome - PASS
```
Comment (chadellison) [inline students/1511/chads_portfolio.md]: Hey @worace,
I changed the 2 to a 3 like you mentioned; I also changed the "provisional pass" to pass. If I was not supposed to do that, just let me know and I can change that back.

**Comment by chadellison** [comment]
@worace I merged master into my branch and updated the pull request. Let me know if you need anything else updated. I appreciate it!

**Comment by worace** [comment]
@chadellison thanks! nice work on the PR & comments

**Pull request merged**